README.md

```diff
@@ -48,16 +48,28 @@ The customers will clone the dev kit using
 They can also simply download the dev kit provided as tar ball from xilinx Alveo Vivado flow [webpage](https://www.xilinx.com/products/boards-and-kits/alveo/u200.html#gettingStarted).
 
 *STEP2*:
+
+setup the software development environment
+
+	source sdk_setup.sh
+	cd sdk
+	sudo ./sdk_install.sh
+
+*STEP3*:
+
+Install the patched xdma driver according to instructions in $ROOT/sdk/xdma_patch/xdma_install.md
```

Comment:
This is only required for cl_dram_dma example or customer designs that use DMA.

Reply:
I pushed a new commit, marking this step as optional.